bazel/README.md

```diff
@@ -260,6 +260,7 @@ for how to update or override dependencies.
 
 1. Install Golang on your machine. This is required as part of building [BoringSSL](https://boringssl.googlesource.com/boringssl/+/HEAD/BUILDING.md)
    and also for [Buildifer](https://github.com/bazelbuild/buildtools) which is used for formatting bazel BUILD files.
+   Make sure you have go version 1.17.5 or later.
```

Comment:
im guessing we can just make this `1.17`

Reply:
`1.17.x` ?